### PR TITLE
Move log to creating place

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

keep seeing 
```
I1113 00:24:14.043296       1 openstackmachine_controller.go:291] controllers/OpenStackMachine "msg"="Creating Machine" "cluster"="capi-openstack-3" "machine"="capi-openstack-control-plane-d2fnl" "namespace"="default" "openStackCluster"="capi-openstack-3" "openStackMachine"="capi-openstack-control-plane-rthx6"

```

but actually it's not creating .. it's getOrCreate, so the log is confusing
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: controllers/openstackmachine_controller.go [hunk 1]
```diff
@@ -288,8 +288,6 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, logger
 		return ctrl.Result{}, err
 	}
 
-	logger.Info("Creating Machine")
```
Comment (hidekazuna) [inline controllers/openstackmachine_controller.go:291]: Not good message is true. I suggest to update to 'Reconciling Machine create started'
in contrast to L370 logger.Info("Reconciled Machine create successfully").

**Comment by hidekazuna** [comment]
It is best to see event. because SuccessfulCreateServer is just after calling createInstance.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/compute/instance.go#L145

**Comment by jichenjc** [comment]
> 
> 
> It is best to see event. because SuccessfulCreateServer is just after calling createInstance.
> https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/compute/instance.go#L145

yes, but reading log makes me very confusing .. so better update log

**Comment by hidekazuna** [comment]
> 
> 
> > It is best to see event. because SuccessfulCreateServer is just after calling createInstance.
> > https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/compute/instance.go#L145
> 
> yes, but reading log makes me very confusing .. so better update log

Currently create or delete resource start message is not logged. You can refer to network resources creation. We need logging policy.

**Comment by jichenjc** [comment]
> > > It is best to see event. because SuccessfulCreateServer is just after calling createInstance.
> > > https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/compute/instance.go#L145
> > 
> > 
> > yes, but reading log makes me very confusing .. so better update log
> 
> Currently create or delete resource start message is not logged. You can refer to network resources creation. We need logging policy.

agree

so are you suggesting should I add in this PR for all those logs ?

**Comment by hidekazuna** [comment]
> > Currently create or delete resource start message is not logged. You can refer to network resources creation. We need logging policy.
> 
> agree
> 
> so are you suggesting should I add in this PR for all those logs ?

Raised another issue.

**Comment by jichenjc** [comment]
recheck

**Comment by jichenjc** [comment]
@hidekazuna so what about this one? maybe merge first? or anything need change ? thanks

**Comment by jichenjc** [comment]
recheck

**Pull request merged**